include/stdexec/execution.hpp

```diff
@@ -3285,10 +3311,7 @@ namespace stdexec {
         __minvocable<__which<__cust_sigs>, _Sender, _Env>;
 
     template <class _Sender, class _Env>
-      using __sender_t = __t<__sender<stdexec::__id<remove_cvref_t<_Sender>>, stdexec::__id<remove_cvref_t<_Env>>>>;
-
-    template <class _Sender, class _Env>
-      using __receiver_t = __t<__receiver<stdexec::__id<remove_cvref_t<_Sender>>, stdexec::__id<remove_cvref_t<_Env>>>>;
+      using __sender_t = __t<__sender<stdexec::__id<_Sender>, stdexec::__id<remove_cvref_t<_Env>>>>;
```

Comment:
This isn't quite right. `_Sender` could be a reference type here. `__id` of a reference type isn't going to do anything useful. You'll need a bit of nastiness here to remove the cvref, apply `__id<>`, then slap the cvref onto the Id, and pass that as a template parameter to `__sender`.

Then within `__sender`, you'll have to undo all of that.

This dance gets done in a number of places. We should probably have some alias templates to make this simpler.

Reply:
Do we need all cvref, or can we remove cv from `_Sender`, then do the same dance but only for the ref-ness?